### DISCUSSION TITLE
Turkish Tranlated file completed. Can you add my translation files

### DISCUSSION THREAD

**Discussion by crasadure** [YerongAI/Office-Tool#642] [General]
I translated your program into Turkish, but I had a problem adding it to the site. If I share the files here, can you add the files?
[Office Tool Turkish Translation Files.zip](https://github.com/YerongAI/Office-Tool/files/11054084/Office.Tool.Turkish.Translation.Files.zip)

**Comment by YerongAI** [comment]
> I translated your program into Turkish, but I had a problem adding it to the site. If I share the files here, can you add the files?
> [Office Tool Turkish Translation Files.zip](https://github.com/YerongAI/Office-Tool/files/11054084/Office.Tool.Turkish.Translation.Files.zip)
> 

Hello, thanks for your great work.

Can you reopen your pull request? We are still in progress to verify your translation, please be patient and wait for us.

We will finish them ASAP.
